content-security-policy/script-src/script-src-strict_dynamic_double_policy_different_nonce.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+    <title>A separate policy with more nonces works correctly with `strict-dynamic` in the script-src directive.</title>
+    <script src='/resources/testharness.js' nonce='dummy'></script>
+    <script src='/resources/testharnessreport.js' nonce='dummy'></script>
+    <script src='../support/checkSecurityPolicyViolations.sub.js' nonce='dummy'></script>
+    <!-- CSP served:
+    1) Content-Security-Policy: script-src 'strict-dynamic' 'nonce-dummy'
+    2) Content-Security-Policy: script-src 'nonce-dummy' 'nonce-dummy2'
+    -->
+</head>
+
+<body>
+    <h1>A separate policy with more nonces works correctly with `strict-dynamic` in the script-src directive.</h1>
+    <div id='log'></div>
+
+    <script nonce='dummy'>
+        async_test(function(t) {
+            var e = document.createElement('script');
+            e.src = 'simpleSourcedScript.js';
+            e.onload = t.unreached_func('OnLoad should not be triggered.');
+            document.body.appendChild(e);
+            t.done();
```

Comment:
Calling `done` here means that you're not giving `onload` time to execute. I think you could be more precise by hooking the `onerror` event instead:

```
e.onload = t.unreached_func();
e.onload = t.step_func_done();
document.body.appendChild(e).
```